taxonomies/ingredients.txt

```diff
@@ -482,7 +482,7 @@ en:emulsifier
 bg:Емулгатор
 cs:Emulgátor
 da:Emulgator
-de:Emulgator
+de:Emulgator, Emulgatoren
```

Comment:
The ingredients that are additives classes can be removed from the ingredients taxonomy, the additives classes taxonomy is automatically added to the ingredients taxonomy.